modules/core/src/main/scala/com.snowplowanalytics.snowplow.databricks/processing/DatabricksUploader.scala

```diff
@@ -123,6 +124,21 @@ object DatabricksUploader {
     case nf: NotFound =>
       // NotFound exception messages are clean, short and helpful
       s"Not Found: ${nf.getMessage}"
+    case e: UnknownHostException =>
+      if (e.getMessage.matches(""".*\bnodename nor servname provided, or not known\b.*"""))
+        "Invalid Databricks hostname"
+      else
+        e.getMessage
```

Comment:
Do you have an example of an error message you might see from line 131?  Asking because I wonder if we can make this simpler, and just always send "Invalid Databricks hostname" for *any* `UknownHostException`. Without the `if` statement.  Or whether `e.getMessage` adds extra context which would otherwise be invisible?